tests/cli/workflow/taxprofiler/test_cli_taxprofiler_metrics_deliver.py

```diff
@@ -54,15 +59,29 @@ def test_metrics_deliver(
     taxprofiler_case_id: str,
     sample_name: str,
     another_sample_name: str,
+    taxprofiler_metrics_deliverables: Path,
 ):
     """Test command with a case id and a finished analysis which should execute successfully."""
     caplog.set_level(logging.INFO)
-    # GIVEN case-id
+
+    # GIVEN case id
+
+    metrics_content: str = "name: filtering_result_passed_filter_reads\n"
 
     result = cli_runner.invoke(metrics_deliver, [taxprofiler_case_id], obj=taxprofiler_context)
 
     # THEN command should execute successfully
     assert result.exit_code == EXIT_SUCCESS
 
+    # THEN metrics deliverable file should be generated
+    assert taxprofiler_metrics_deliverables.is_file()
+
+    # THEN assert metrics_content in metrics_deliverables
+    with open(taxprofiler_metrics_deliverables, "r") as file:
```

Comment:
Use the YAML read function from the IO module to keep it DRY.

Reply:
Added [here](https://github.com/Clinical-Genomics/cg/blob/14e7e092ed6a7b237cdf5ad6d19e51ba2a04ba66/tests/cli/workflow/taxprofiler/test_cli_taxprofiler_metrics_deliver.py#L73)